testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -86,6 +86,21 @@ public void testAccessControlInsert()
         assertThat(aliceExecutor.executeQuery(format("SELECT * FROM %s", tableName))).hasRowsCount(1);
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlUpdate()
```

Comment:
are there any engine level tests that we can add?

Reply:
Maybe we could add some tests to `io.trino.execution.TestGrantOnTable`?

Reply:
Added UPDATE privilege to `io.trino.execution.TestGrantOnTable` and `io.trino.execution.TestRevokeOnTable`